src/core.js

```diff
@@ -442,15 +450,9 @@ jQuery.extend( {
 	support: support
 } );
 
-// JSHint would error on this code due to the Symbol not being defined in ES5.
```

Comment:
I would prefer this way, since i fear someone might introduce some debug variable or something.

But if you feel strongly about i'm not gonna object